js/web/lib/wasm/jsep/init.ts

```diff
@@ -291,6 +291,8 @@ export const init = async (
       },
       // jsepDownloadTensor
       async (tensorId: number, dstBuffer: ArrayBufferView | ArrayBuffer) => backend.downloadTensor(tensorId, dstBuffer),
+      // jsepEnableTraceEvent
+      env.trace,
```

Comment:
you can use `!!env.trace` to convert it to bool

Reply:
Thanks, that's better for initialization.